### ISSUE TITLE
The PERF rules for ruff make this plugin crash (version 1.0.7)

### ISSUE THREAD

**Issue by fvalverd** [sublimelsp/LSP-ruff#26]
If you want to use the PERF rules for ruff, this plugin crashes with the following error:
```
LSP-ruff: error: TOML parse error at line 70, column 5
   |
70 |     "PERF",  #  Perflint
   |     ^^^^^^
Unknown rule selector: `PERF`
```

I checked the versions of `ruff` and `ruff-lsp` for this plugin and they are outdated, the current versions are:
```
ruff==0.0.272
ruff-lsp==0.0.32
```
and so far the latest versions are:
```
ruff==0.0.277
ruff-lsp==0.0.35
```

I manually updated the previous libraries at `~/.cache/sublime-text/Package Storage/LSP-ruff` and the PERF rules worked without any problem.

There is already a dependabot PR for upgrading them at https://github.com/sublimelsp/LSP-ruff/pull/25, but those are not the latest either.

**Comment by LDAP** [comment]
Merged #25 and published a new release which should show up on Package Control shortly.

We follow the version of ruff that ruff-lsp uses for compatibility reasons, see here:
https://github.com/astral-sh/ruff-lsp/blob/43e60ebf060a924a205349920d501f57bfab1a44/pyproject.toml#L39.

**Comment by fvalverd** [comment]
Thank you! I think with that should be enough to fix the crash.

From my side I will keep updating the `ruff` version manually, because the releases are very frequent with bugfixes and features https://github.com/astral-sh/ruff/releases
